File: Cursos/POO/Programación Orientada a Objetos POO/Code/JAVA/UberPool.java
// herencia de la clase Car
class UberPool extends Car {
    String brand;
    String model;

    public UberPool(String license, Account driver, String brand, String model) {
        // Super hace referencia a la clase padre (Car)
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
}

File: Cursos/POO/Programación Orientada a Objetos POO/Code/PHP/UberPool.php
<?php
require_once('Car.php');

class UberPool extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model) {
        // paent hace referencia a la clase padre
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
}
?>